site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -87,6 +87,22 @@ kubectl create -f gameserver.yaml
 kubectl apply -f gameserver.yaml
 ```
 
+{{< alert title="Note" color="info">}}
+If you made changed `main.go` again and want to apply the changes to the new game servers, then you also need 
+  to modify the `gamerserver.yaml` file's `imagePullPolicy` to be `Always`, or else the new game servers might
+  only pull a cached version of the image, which doesn't have the new changes.
```

Comment:
I think a link to https://kubernetes.io/docs/concepts/containers/images/#updating-images would be good. 

Something like "See the documentation for ImagePullPolicy" for more details.

Reply:
Done